pom.xml

```diff
@@ -69,7 +69,7 @@
             <dependency>
               <groupId>com.puppycrawl.tools</groupId>
               <artifactId>checkstyle</artifactId>
-              <version>8.26</version>
+              <version>8.29</version>
```

Comment:
Release Notes: The vulnerabilities on first glance don't seem to affect us a great deal, also no backward compatibility issues reported in the notes